src/fides/api/alembic/migrations/versions/xx_2025_10_17_1603-5093e92e2a5a_add_consent_data_v3_to_the_database.py

```diff
@@ -0,0 +1,54 @@
+"""Add consent data v3 to the database
+
+Revision ID: 5093e92e2a5a
+Revises: a55e12c2c2df
+Create Date: 2025-10-17 16:03:37.933367
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "5093e92e2a5a"
+down_revision = "a55e12c2c2df"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "privacy_preferences",
```

Comment:
We should make these table names singular so we follow the current convention in Fides